.github/workflows/cypress.yml

```diff
@@ -0,0 +1,31 @@
+on:
+  push:
+    branches:
+      - e2e/55-default-admin-settings
```

Comment:
The test is passing, but we need to update this run condition before merging.

Reply:
yes absolutely, this is just so that I can ensure things are working well on Github Actions